bats/tests/helpers/vm.bash

```diff
@@ -267,7 +275,10 @@ wait_for_container_engine() {
     trace "waiting for api /settings to be callable"
     try --max 30 --delay 5 rdctl api /settings
 
-    run jq_output .containerEngine.allowedImages.enabled
+    trace "waiting for VM to be available"
+    wait_for_shell
```

Comment:
I wonder if the `wait_for_shell` call shouldn't be moved into `wait_for_service_status`, because that is the function that needs the shell to be available.

Just so it does the right thing when `wait_for_service_status` is called from other places.

Reply:
Done. This also means I can undo the `jq_output` change.